src/VirtoCommerce.Platform.Data/ExportImport/PlatformExportImportManager.cs

```diff
@@ -147,137 +147,152 @@ private async Task ImportPlatformEntriesInternalAsync(ZipArchive zipArchive, Pla
             var batchSize = 20;
 
             var platformZipEntries = zipArchive.GetEntry(PlatformZipEntryName);
-            if (platformZipEntries != null)
+            if (platformZipEntries is null || !manifest.HandleSecurity)
```

Comment:
`manifest.HandleSecurity` is not arbitrary for all cases, settings check `manifest.HandleSettings`.
I'd make a standalone ticket and PR for this ticket, as it should be tested carefully.

Reply:
Reverted, fixed, moved to another PR https://github.com/VirtoCommerce/vc-platform/pull/2050